test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -483,6 +486,44 @@ TEST(QuicPlatformTest, QuicTestOutput) {
                       QuicRecordTestOutput("quic_test_output.3", "output 3 content\n"));
 }
 
+class QuicMemSliceTest : public QuicTest {
+public:
+  QuicMemSliceTest() {
+    slice_ = quic::QuicMemSlice(
```

Comment:
where is QuicMemSlice defined? I can't find it in this PR or in a recently synced git envoy client.

Reply:
It's in external quiche code: https://quiche.googlesource.com/quiche/+/refs/heads/master/quic/platform/api/quic_mem_slice.h.
All quiche code is brought in via https://github.com/envoyproxy/envoy/blob/a4507773353456f0a6228dee9fe1b9836cd36562/bazel/external/quiche.genrule_cmd